job-server/src/spark.jobserver/JobManager.scala

```diff
@@ -0,0 +1,63 @@
+package spark.jobserver
+
+import java.io.File
+
+import akka.actor.{AddressFromURIString, Address, Props, ActorSystem}
+import akka.cluster.Cluster
+import com.typesafe.config.{ConfigValueFactory, ConfigFactory, Config}
+import ooyala.common.akka.actor.ProductionReaper
+import ooyala.common.akka.actor.Reaper.WatchMe
+import org.slf4j.LoggerFactory
+import spark.jobserver.io.{JobDAOActor, JobDAO}
+
+/**
+ * Created by ankits on 4/7/15.
+ */
```

Comment:
I don't see the rest of the codebase doing this so to keep in style with that I'd remove it. You'd be in the git history ;)